src/main/java/dal/fatih/todoly/TaskRepository.java

```diff
@@ -0,0 +1,186 @@
+        createTablePreparedStatement = connection.prepareStatement("CREATE TABLE IF NOT EXISTS tasks"
+                + "  (Id INT AUTO_INCREMENT primary key NOT NULL,"
+                + "   taskId UUID(36),"
+                + "   title VARCHAR(35),"
+                + "   description VARCHAR(250),"
+                + "   dueDate DATE)");
+        createTable();
+        createTaskPrepareStatement =
+                connection.prepareStatement("INSERT INTO TASKS " + "VALUES (?,?,?,?,?)");
+        listAllTaskPrepareStatement =
+                connection.prepareStatement("select ID,TASKID,TITLE,DESCRIPTION,DUEDATE from TASKS");
+        getDetailsPreparedStatement =
+                connection.prepareStatement("SELECT taskId,title,description,duedate FROM tasks WHERE taskId=? limit 1");
+        deleteTaskByIdPreparedStatement =
+                connection.prepareStatement("delete FROM tasks WHERE taskId=?");
+        listBetweenPreparedStatement =
+                connection.prepareStatement("select * from TASKS where DUEDATE BETWEEN now() and ?");
+        filterByTitlePreparedStatement =
+                connection.prepareStatement("select taskId,title,description,dueDate from TASKS where title like ? "
+                        + "or description like ?");
+    }
+
+    public void createTable() {
+        try {
+            createTablePreparedStatement.executeUpdate();
+        } catch (SQLException exception) {
+            System.out.println("Check database connections, drivers!!!");
+        }
+    }
+
+    public void create(Task task) {
+        try {
+            createTaskPrepareStatement.setString(1, null);
+            createTaskPrepareStatement.setString(2, task.getId().toString());
+            createTaskPrepareStatement.setString(3, task.getTitle());
+            createTaskPrepareStatement.setString(4, task.getDescription());
+            createTaskPrepareStatement.setString(5, task.getDate().toString());
+            createTaskPrepareStatement.executeUpdate();
+
+            System.out.println("\n" + task.getId() + " Task added");
+        } catch (Exception e) {
+            System.out.println(e.getMessage());
+        }
+    }
+
+    public void listAll() {
+        int counter = 0;
+        try {
+            resultSet = listAllTaskPrepareStatement.executeQuery();
+            while (resultSet.next()) {
+                counter++;
+                String taskId = resultSet.getString(2);
+                String title = resultSet.getString(3);
+                System.out.println("Id: " + taskId);
+                System.out.println("Title: " + title);
+                System.out.println("-------------------------------------------------");
+            }
+            if (counter < 1) {
+                System.out.println("------------------");
+            }
+        } catch (Exception e) {
+            System.out.println(e.getMessage());
+        }
+    }
+
+    public void getDetails(String taskIdInput) {

```

Comment:
Don't do the print here, make this return `Task`

```suggestion
    public Task get(String taskIdInput) {
```